### PR TITLE
Fixed #26607 -- Added InlineModelAdmin.get_formset_initial_data().

### PR DESCRIPTION
**Pull request by dsanders11**

### CODE REVIEW

File: docs/ref/contrib/admin/index.txt [hunk 1]
```diff
@@ -2210,6 +2210,20 @@ The ``InlineModelAdmin`` class adds:
     inline forms. For example, this may be based on the model instance
     (passed as the keyword argument ``obj``).
 
+.. method:: InlineModelAdmin.get_formset_initial_data(request)
+
+    .. versionadded:: 1.10
```
Comment (berkerpeksag) [inline docs/ref/contrib/admin/index.txt]: 1.10 -> 1.11

File: docs/releases/1.10.txt [hunk 2]
```diff
@@ -69,6 +69,10 @@ Minor features
 * Added "No date" and "Has date" choices for ``DateFieldListFilter`` if the
   field is nullable.
 
+* The new :meth:`InlineModelAdmin.get_formset_initial_data()
```
Comment (berkerpeksag) [inline docs/releases/1.10.txt]: Can you please move this to `docs/releases/1.11.txt`?

File: tests/admin_views/admin.py [hunk 3]
```diff
@@ -841,13 +841,21 @@ def view_on_site(self, obj):
...
+    extra = 0
 
     def view_on_site(self, obj):
         return '/worker_inline/%s/%s/' % (obj.surname, obj.name)
 
 
+class ExtraWorkerInlineAdmin(WorkerInlineAdmin):
+    extra = 1
+
+    def get_formset_initial_data(self, request):
```
Comment (berkerpeksag) [inline tests/admin_views/admin.py]: It would be nice to add a test case where `extra != len(get_formset_initial_data)`.

**Comment by berkerpeksag** [comment]
In commit message: Refs -> Fixed

**Comment by berkerpeksag** [comment]
`ticket_26607` needs to be rebased onto latest master.

**Comment by timgraham** [comment]
Closing due to inactivity and per [my comment on the ticket](https://code.djangoproject.com/ticket/26607#comment:3) that suggests adding a more general hook.

**Comment by danihodovic** [comment]
@timgraham was there any progress on the more general hook?

**Comment by felixxm** [comment]
@danihodovic No (as can be checked on the ticket).

**Comment by danihodovic** [comment]
If one was to contribute a "general hook", how would said hook be different than the implementation in this PR?

**Comment by felixxm** [comment]
It's clearly stated in [Tim's comment](https://code.djangoproject.com/ticket/26607#comment:3). Please check a ticket.

**Pull request closed without merging**